pkg/csi/cinder/mount/mount.go

```diff
@@ -238,3 +238,7 @@ func (m *Mount) GetInstanceID() (string, error) {
 	}
 	return "", err
 }
+
+func IsCorruptedMnt(err error) bool {
```

Comment:
I guess I'm a bit confused as to why we wrap this rather than use it directly.

Reply:
@ramineni is trying to clean it up
https://github.com/kubernetes/cloud-provider-openstack/pull/504